src/node_modules/logger.js

```diff
@@ -71,7 +68,7 @@ const requirePeer = codependency.register(module);
 
 const nonConfigurable = (target, name, descriptor) => {
 	descriptor.configurable = false;
-	descriptor.enumerable = true;
+	descriptor.enumerable   = true;
```

Comment:
This is to align assignment operator with previous assignment line. More readable code.